doc/tutorials/unitary_event_analysis.ipynb

```diff
@@ -117,7 +119,7 @@
     "    'suptitle': None,\n",
     "}\n",
     "\n",
-    "\n",
+    "@trials_to_list_of_spiketrainlist\n",
```

Comment:
- use Viziphant function here (later)
- `plot_ue` should accept trial object (-> later in Viziphant)
- update `plot_ue` function in Viziphant
- copy updated code into this notebook for now